templates/angular/ts/editors/default/files/src/app/components/__path__/__name__.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+
+@Component({
+	selector: "editors",
+	template: `
+		<$(widget) [(options)]="options" widgetId='$(widget)'></$(widget)>
+	`
+})
+export class $(ClassName)Component {
+	public options: Ig$(ClassName);
```

Comment:
Note that class name used like this will only work when the default name is selected for the template and gen wrong interface on most user input. A separate template variable based on the control name would be required